### DISCUSSION TITLE
dyld: Library not loaded: @rpath/WeatherClient.framework/WeatherClient 0114-designing-dependencies-pt5

### DISCUSSION THREAD

**Discussion by saroar** [pointfreeco/episode-code-samples#102] [Q&A]
remove framework for view and add the swift package for view and getting this error on a real device 

Apple Swift version 5.3.2 (swiftlang-1200.0.45 clang-1200.0.32.28)
Target: x86_64-apple-darwin20.3.0
XCODE: Version 12.4 (12D4e)

```
dyld: Library not loaded: @rpath/WeatherClient.framework/WeatherClient
  Referenced from: /private/var/containers/Bundle/Application/714B6D18-5451-4ED0-B753-87050889ED83/DesigningDependencies.app/Frameworks/WeatherFeature.framework/WeatherFeature
  Reason: image not found
dyld: launch, loading dependent libraries
DYLD_LIBRARY_PATH=/usr/lib/system/introspection
DYLD_INSERT_LIBRARIES=/Developer/usr/lib/libBacktraceRecording.dylib:/Developer/usr/lib/libMainThreadChecker.dylib:/Developer/Library/PrivateFrameworks/DTDDISupport.framework/libViewDebuggerSupport.dylib
```

[DesigningDependencies.zip](https://github.com/pointfreeco/episode-code-samples/files/6089018/DesigningDependencies.zip)

**Comment by stephencelis** [comment]
Hi @saroar! Thanks for starting a convo. Our episode code unfortunately can't always be so robust as to work on devices, but we welcome discussion for those that want to troubleshoot the problems and share solutions. I'm going to convert this to a discussion now. Thanks for posting!
